app/views/journal_feeds/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<body class="vsc-initialized">
+<main role="main" class="container">
+  <div class="row row-offcanvas row-offcanvas-right">
+    <div class="col-12 col-md-9">
+      <h2>Abstracts</h2>
+        <% @journal_feeds.each do |journal_feed| %>
+          <% current_user.subscriptions.each do |subscription| %>
+            <% if subscription.user_id == current_user.id %>
+              <% journal_feed.journals.each do |journal| %>
+                <% journal.abstracts.each do |abstract| %>
+                  <br>
+                  <h4> <%= link_to "#{abstract.title}", abstract.url %> </h4>
```

Comment:
You don't need to create a string here. Since `abstract.title` already is a string, you can just do `link_to abstract.title, abstract.url`